server/jetty/README.md

```diff
@@ -1,5 +1,44 @@
 # Native packaging for Deephaven Jetty server
 
+### Setting up a Python virtual environment
+
+This is an optional prerequisite, but lets the later commands to run the server work without
+specifying groovy as the console language. If you skip this step, be sure to use groovy, or
+there will be errors on startup indicating that the Python environment is not suitable.
+
+See https://github.com/deephaven/deephaven-core/issues/1657 for more discussion
+
+1. On MacOS there is an extra patch to apply at this time, to add an extra argument to clang,
```

Comment:
On Fedora, I also needed python development headers installed. That looked something like

```
sudo dnf install python-devel
```

for me. Likely want a note about headers, and how to get them for ubuntu, fedora, maybe others?

Reply:
Probably, but better put it #1657 if we require that, this isn't meant to be exhaustive at this time.